src/file_pipe_log.rs

```diff
@@ -144,8 +170,8 @@ impl LogManager {
 
         let mut path = PathBuf::from(&self.dir);
         path.push(generate_file_name(self.active_file_id, self.name_suffix));
-        let fd = Arc::new(LogFd(open_active_file(&path)?));
-        let bytes = write_file_header(fd.0)?;
+        let (fd, bytes) = create_active_file(&path)?;
+        let fd = Arc::new(fd);
 
         self.active_log_size = bytes;
         self.active_log_capacity = bytes;
```

Comment:
Ideally, we could use a `ReadableFile` for inactive files, and `RandomAccessFile` for active file, and put `active_log_size` and `active_log_capacity` inside. Maybe as a followup task.

Reply:
Agree.

Reply:
I will leave the `RandomAccessFile` work in the next pr, this pr just refines the version format.